Src/Utils/Paths.ts

```diff
@@ -1,14 +1,17 @@
-import finder from "find-package-json";
+import fs from "fs";
 import path from "path";
 
-// crawl up parent directories until we find the package.json of the calling software
-let lRootDir: string = process.cwd();
-for (const lPackageJson of finder())
+const DirHasGulpfile = (aDir: string): boolean => fs.readdirSync(aDir)
+    .some((aFile: string) => /^[Gg]ulpfile\.[tj]s$/.test(path.basename(aFile)));
```

Comment:
Instead of reading the name of all files in a dir, would you be able to check for the existence of one of the following files:
 - Gulpfile.js
 - Gulpfile.ts

That should be faster and safer (constant time as opposed to variable on the number of files in a dir)

Note, this would mean we don't accept lower case gulpfiles which is something we should document on the readme if we're okay with that, else we'd have to check for the existence of 4 possible files:
 - Gulpfile.ts
 - Gulpfile.js
 - gulpfile.ts
 - gulpfile.js